python/lsst/dax/distribution/DataGenConnection.py

```diff
@@ -582,7 +588,8 @@ def testMethods(self):
         if len(diff) != 0:
             print("large range chunkMsg failed", diff)
             success = False
-        if success is None: success = True
+        if success is None:
+            success = True
```

Comment:
That's a special case of success, which not expected to happen often?, so a print message here could help to track its occurrence.

Reply:
This isn't a special occurrence, this is just me being uptight about saying something is successful before it is actually successful.